timemachine/fe/rbfe.py

```diff
@@ -98,10 +99,18 @@ def assert_all_states_have_same_masses(initial_states: List[InitialState]):
     np.testing.assert_array_almost_equal(deviation_among_windows, 0, err_msg="masses assumed constant w.r.t. lambda")
 
 
+@dataclass
+class Host:
+    system: VacuumSystem
+    masses: List[float]
```

Comment:
[nonblocking, unrelated to fixing test]

Possibly rename to `physical_masses`, or have 2 fields `physical_masses` and `hmr_masses`? (Current usages look correct, but good to preempt accidentally applying HMR 0 or 2 times.)

Reply:
Ah, I thought we currently disambiguated by prefixing HMR masses with `hmr_`. (I'm not immediately seeing where this field is used for HMR masses). Is the idea just to be extra explicit?

Reply:
Sorry, this is tangential -- will double-check this separately!

>Ah, I thought we currently disambiguated by prefixing HMR masses with hmr_. (I'm not immediately seeing where this field is used for HMR masses)

Good point -- the current convention `masses` vs `hmr_masses` is unambiguous already!

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/966/commits/271007e6c96ba80734743eda7d2a96e1d1309bfe